.github/workflows/publish.yml

```diff
@@ -3,7 +3,7 @@ name: publish
 on:
   workflow_run:
     workflows: [ci]
-    branches: [main, v2, v1-hotfix]
```

Comment:
May want to keep v1-hotfix here?

Reply:
I don't think so. Here is what I want to achieve:

- when the `main` branch finishes `ci` workflow, trigger `publish.yml`, which will git pull the default branch (`main`) and run `npm release`.
- when the `v1-hotfix` branch finishes its `ci` workflow, trigger `publish-v1.yml`, which will git pull the `v1-hotfix` branch and run `npm release`.

Reply:
Of course! 🤦‍♂️